packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -520,19 +568,22 @@ class MultiFrameImageStreamCompleter extends ImageStreamCompleter {
     _framesEmitted += 1;
   }
 
-  bool get _hasActiveListeners => _listeners.isNotEmpty;
```

Comment:
I'm not sure what is the scenario we're fixing. Can you describe the issue in the PR description for posterity?

Reply:
This fix was bad. I've changed the fix to match the PR description.